test/handler.cc

```diff
@@ -52,6 +52,8 @@ void Handler::onAttrData(const ptr_lib::shared_ptr<const Interest>& interest, co
     }
     else{
       cerr << "Protobuf decoding error" << endl;
+      string dir_html((char*)content.buf(), content.size());
```

Comment:
What this and next line for? If you want additional info to error message, it should got to std::cerr and be properly explained